recipes/FastTree/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: fasttree
+  version: "2.1.8"
+
+source:
+  fn: FastTree-2.1.8.c
+  url: http://www.microbesonline.org/fasttree/FastTree-2.1.8.c
+
+build:
+  number: 0
+  script:
+    - mkdir -p "$PREFIX/bin"                                  # [osx or linux]
+
+    - gcc -DUSE_DOUBLE -O3 -finline-functions -funroll-loops -Wall -o FastTree FastTree-2.1.8.c -lm
+    - gcc -fopenmp -DUSE_DOUBLE -O3 -finline-functions -funroll-loops -Wall -o FastTreeMP FastTree-2.1.8.c -lm
```

Comment:
This line usually starts `-DOPENMP -fopenmp`. I've omitted `-DOPENMP` to make it compile using the default OSX gcc. idk if's true OpenMP or not.